matrix-synapse-riot/terraform/matrix-server/bootstrap-cleanup.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+# Disable root login in sshd_config
+sed -i -e "s/PermitRootLogin yes/PermitRootLogin no/g" /etc/ssh/sshd_config
+
+# Clean up the tmp directory
+shred -u /tmp/matrix-server/*
```

Comment:
Does this get rid of the folder?

Reply:
added rm -rf  /tmp/matrix-server/ under that.